docs/design/bulk-import.drawio

```diff
@@ -0,0 +1,86 @@
+      <root>
+        <mxCell id="0" />
+        <mxCell id="1" parent="0" />
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-25" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;" edge="1" parent="1" source="3NLjgXGCndHzrWQDtMyP-1" target="3NLjgXGCndHzrWQDtMyP-8">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-1" value="" style="sketch=0;points=[[0,0,0],[0.25,0,0],[0.5,0,0],[0.75,0,0],[1,0,0],[0,1,0],[0.25,1,0],[0.5,1,0],[0.75,1,0],[1,1,0],[0,0.25,0],[0,0.5,0],[0,0.75,0],[1,0.25,0],[1,0.5,0],[1,0.75,0]];outlineConnect=0;fontColor=#232F3E;fillColor=#E7157B;strokeColor=#ffffff;dashed=0;verticalLabelPosition=bottom;verticalAlign=top;align=center;html=1;fontSize=12;fontStyle=0;aspect=fixed;shape=mxgraph.aws4.resourceIcon;resIcon=mxgraph.aws4.sqs;" vertex="1" parent="1">
+          <mxGeometry x="200" y="440" width="40" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-26" style="edgeStyle=orthogonalEdgeStyle;rounded=1;orthogonalLoop=1;jettySize=auto;html=1;curved=0;" edge="1" parent="1" source="3NLjgXGCndHzrWQDtMyP-2" target="3NLjgXGCndHzrWQDtMyP-1">
+          <mxGeometry relative="1" as="geometry">
+            <Array as="points">
+              <mxPoint x="100" y="460" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-28" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;" edge="1" parent="1" source="3NLjgXGCndHzrWQDtMyP-2" target="3NLjgXGCndHzrWQDtMyP-17">
+          <mxGeometry relative="1" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-2" value="" style="image;points=[];aspect=fixed;html=1;align=center;shadow=0;dashed=0;image=img/lib/allied_telesis/computer_and_terminals/Personal_Computer.svg;" vertex="1" parent="1">
+          <mxGeometry x="85" y="560" width="30" height="40.66" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-3" value="Client application" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" vertex="1" parent="1">
+          <mxGeometry x="70" y="600.66" width="60" height="40" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-4" value="Bulk import queue takes a job with a list of input files" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" vertex="1" parent="1">
+          <mxGeometry x="170" y="380" width="100" height="60" as="geometry" />
+        </mxCell>
+        <mxCell id="3NLjgXGCndHzrWQDtMyP-6" value="Spark cluster in EMR receives job, sorts data and adds it to a Sleeper table" style="text;html=1;align=center;verticalAlign=middle;whiteSpace=wrap;rounded=0;" vertex="1" parent="1">
```

Comment:
Can we change this to "partitions and sorts data" rather than "sorts data"?

Reply:
Done